internal/wasm/sdk/opa/errors/errors.go

```diff
@@ -23,4 +23,6 @@ var (
 	ErrNonBoolean = errors.New("non-boolean decision")
 	// ErrInternal is the error returned if the evaluation fails due to an internal error.
 	ErrInternal = errors.New("internal error")
+	// ErrCancelled is the error returned if the evaluation is cancelled.
+	ErrCancelled = errors.New("cancelled")
```

Comment:
I noticed these errors make it all the way out to the user. It would be nice to (eventually, not in this PR) unify them a bit w/ the errors from topdown:

```
(sr/wasm/context)torin:~/go/src/github.com/open-policy-agent/opa$ ./opa_linux_amd64 test --timeout=1s ~/x.rego -v
data.x.test_xxx: ERROR (1.0009364s)
  eval_cancel_error: numbers.range: timed out before generating all numbers in range
--------------------------------------------------------------------------------
ERROR: 1/1
(sr/wasm/context)torin:~/go/src/github.com/open-policy-agent/opa$ ./opa_linux_amd64 test --timeout=1s ~/x.rego -t wasm
data.x.test_xxx: ERROR (1.0010134s)
  cancelled
--------------------------------------------------------------------------------
ERROR: 1/1
```

Reply:
Definitely! I'll create an issue to track this.

Reply:
OK I didn't exactly cover myself in glory with the issue, but here it is: https://github.com/open-policy-agent/opa/issues/3225